### ISSUE TITLE
distill loss weight?

### ISSUE THREAD

**Issue by yuzuda283** [ZhangXInFD/SpeechTokenizer#8]
I can't find the distill loss weight in paper. I try to copy the distillation experiment on dac,  the reconstruction seems normal but the first codebook seems doesn't  disentangle the semantic information.

**Comment by ZhangXInFD** [comment]
We have published our trainning code and training config in the repository. You can check the distill loss weight in corresponding file. BTW, we meet the same question when we attempt to do the distillation experiment on dac. It seems dac's VQ is not very compatible with the semantic distillation task.

**Comment by UkiTenzai** [comment]
这可能是因为编码器输出的feature本身是离散值，不包含前面的网络层的计算图，所以反传distill loss实际上可能只训练了用于对齐的全连接层（model.py第38行的self.transform = nn.Linear(config.get('dimension'), config.get('semantic_dimension'))）
